hw/lint/tools/veriblelint/rules.vbl

```diff
@@ -17,3 +17,5 @@ parameter-name-style=localparam_style:CamelCase|ALL_CAPS
 
 # we allow nested struct definitions
 -typedef-structs-unions
+
+-generate-label-prefix
```

Comment:
Why is this change needed? Is it part of the Verible version update (if so, let's tackle it in a separate PR or add to #6434) or a requirement for the parsing code?

Reply:
When I bumped Verible version the linter in CI has been complaining that some Opentitan code violates this (newly introduced) rule. This seems to be fixed now.